### PR TITLE
chore: migrate goerli to sepolia

### PR DESCRIPTION
**Pull request by dcposch**

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/link/NoteScreen.tsx [hunk 1]
```diff
@@ -144,7 +140,7 @@ function NoteDisplayInner({
   const nonce = useMemo(() => new DaimoNonce(nonceMetadata), [ephemeralOwner]);
 
   const isV2RecipientClaim =
-    noteStatus.contractAddress === daimoEphemeralNotesV2Address &&
+    noteStatus.link.type === "notev2" &&
```
Comment (nalinbhardwaj) [inline apps/daimo-mobile/src/view/screen/link/NoteScreen.tsx:143]: 👍 these are actually not equivalent (there were two migrations contract v1 > v2 and link v1 > v2 independently) but its fine.

File: apps/daimo-web/src/components/AppOrWalletCTA.tsx [hunk 2]
```diff
@@ -244,22 +247,24 @@ async function linkStatusToAction(
         urlHash
       );
 
-      if (contractAddress === daimoEphemeralNotesV2Address) {
+      if (linkStatus.link.type === "notev2") {
```
Comment (nalinbhardwaj) [inline apps/daimo-web/src/components/AppOrWalletCTA.tsx:250]: same as above

File: apps/scratchpad/src/createAccount.ts [hunk 3]
```diff
@@ -131,7 +131,10 @@ export async function createAccount() {
   const bundlerClient = getBundlerClientFromEnv();
   const sender: OpSenderCallback = async (op: UserOpHex) => {
     const vc = new ViemClient(publicClient, publicClient, walletClient);
-    return bundlerClient.sendUserOp(op, vc);
+    const hash = bundlerClient.getOpHash(op, vc.publicClient);
+    console.log(`NOT sending userOp. Hash: ${hash}`);
+    return hash;
+    // return bundlerClient.sendUserOp(op, vc);
```
Comment (nalinbhardwaj) [inline apps/scratchpad/src/createAccount.ts:137]: TODO

Comment (dcposch) [reply apps/scratchpad/src/createAccount.ts:137]: it's the scratchpad 🤷‍♂️

**Review by nalinbhardwaj — APPROVED** [review]
lgtm broadly, feel free to merge if everything works so can rebase other contracts and start testing on sepolia

File: packages/contract/src/DaimoEphemeralNotes.sol [hunk 4]
```diff
@@ -27,8 +27,7 @@ contract DaimoEphemeralNotes {
     event NoteRedeemed(Note note, address redeemer);
 
     // Stablecoin token to use for notes, or 0x0 for USDC.
-    // This allows us to CREATE2-deploy this contract at the same address
-    // on Base and Base Goerli.
+    // This allows us to CREATE2-deploy at the same address on different chains.
```
Comment (nalinbhardwaj) [inline packages/contract/src/DaimoEphemeralNotes.sol:30]: this edit changes the create2 address no, would revert unless you're migrating deployment

Comment (dcposch) [reply packages/contract/src/DaimoEphemeralNotes.sol]: good call, reverted

**Pull request merged**